web/src/components/demo_pages/Intro/intro.css

```diff
@@ -0,0 +1,91 @@
+@import url('../../vars.css');
+
+.container {
+    height: 100vh;
+    width: 100vw;
+    padding-top: 2.5vh;
+    background: linear-gradient(180deg, #f3f2f1 0%, #ffffff 101.23%);
+}
+
+.text-box {
+    display: flex;
+    flex-direction: column;
+    padding: 0px;
+    height: 173px;
+    left: calc(50% - 700px / 2 + 1px);
+    bottom: 68px;
```

Comment:
So generally speaking I would avoid specifying absolute pixel values like this. When it comes to heights, you don't know how long strings will be when localized into other languages, so you'll want to account for fluid text lengths in your layout. And since this demo will be available on desktops as well rather than just set tablet sizes, you'll want this to be fluid. While in this case the overflow from the container gets displayed anyway because there's nothing else around/underneath, in more complex layouts the fact that the button pushes beyond the bounds of the text-box at different widths can become a problem. 

I'm also not clear on what the `bottom` value is accomplishing here, or why you're using calculated positioning instead of just centering the text? A thing you'll also want to think about is how this will look in languages that are RTL, so if you're specifying positions based on their relationship to a starting point on the left that can have unintended consequences.